src/index.test.js

```diff
@@ -12,16 +13,16 @@ jest.mock('./ccpa', () => ({
 	},
 }));
 
-jest.mock('./tcfv2', () => ({
-	TCFv2: {
-		init: jest.fn(),
-		showPrivacyManager: jest.fn(),
-		willShowPrivacyMessage: () => Promise.resolve('iwillshowit'),
-	},
-}));
+const TCFv2 = {
```

Comment:
maybe this should be called `TCFv2Spy` and then the import can be the default name?

Reply:
I don’t mind either way, but it made more sense to do it this way to make it work the same way as mocks, meaning you’re calling a function with the same name as you would normally.

Calling `TCFv2.init` behaves as you would expect, as it is calling `actualTCFv2.init`.